frontend/src/views/Systems.vue

```diff
@@ -266,6 +195,13 @@ export default Vue.extend({
           command: command
         })
       });
+    },
+    isConnected(lastSeen: string): boolean {
+      let diff = Number(new Date()) - Number(new Date(lastSeen));
+      if (diff > 14999) { // arbitrary number in ms
+        return false;
+      }
+      return true;
```

Comment:
Comparison result can be returned directly.

```suggestion
     return diff < 70 * 1000;
```